cmd/observer/node_builder/observer_builder.go

```diff
@@ -1187,6 +1332,19 @@ func (builder *ObserverServiceBuilder) enqueueRPCServer() {
 		builder.RestMetrics = m
 		return nil
 	})
+	builder.Module("transaction timing mempools", func(node *cmd.NodeConfig) error {
+		var err error
+		builder.CollectionsToMarkFinalized, err = stdmap.NewTimes(50 * 300) // assume 50 collection nodes * 300 seconds
```

Comment:
I don't think we need to support tx tracking on observers since they will only ever receive the data once the block is sealed. can you create a noop version of the tracker and pass it instead